src/test/func/pagemap/pagemap.cc

```diff
@@ -99,6 +101,20 @@ void test_pagemap(bool bounded)
 
   // Check pattern is correctly stored
   std::cout << std::endl;
+
+  if (bounded)
+  {
+    // Check we have not corrupted the heap.
+    for (address_t ptr = low; ptr < high; ptr++)
+    {
+      auto* p = (char*)ptr;
```

Comment:
CHERI CI strikes again here, too; perhaps `low` wants to be a `uintptr_t` rather than an `address_t`?